samples/quickstart.js

```diff
@@ -37,8 +37,25 @@ async function quickstart(
       branchName,
     },
   };
-  await cb.runBuildTrigger(request);
+  const [resp] = await cb.runBuildTrigger(request);
   console.info(`triggered build for ${triggerId}`);
+  const [build] = await resp.promise();
+
+  const STATUS_LOOKUP = [
+    'UNKNOWN',
+    'Queued',
+    'Working',
```

Comment:
it would be nice if the generator stubbed out a functional example of using the library with TypeScript, e.g,. how to import the class, instantiate it.

Reply:
Should be taken care of by Samplegen, but unit tests (which are in TypeScript) actually give some idea about how to import and use the library.

Reply:
I think this will be also covered by system tests issue https://github.com/googleapis/gapic-generator-typescript/issues/89.